src/rust/ide/view/graph-editor/src/component/breadcrumbs.rs

```diff
@@ -541,6 +542,18 @@ impl Breadcrumbs {
             // === Relayout ===
 
             eval_ scene.frp.camera_changed(model.camera_changed());
+
+            eval model.project_name.frp.output.width ((width) {
+                model.relayout_for_project_name_width(*width)
+            });
+
+
+            // === Pointer style ===
+
+            eval model.project_name.frp.output.pointer_style ((style)
+                frp.outputs.pointer_style.emit(style);
+            );
```

Comment:
One request here - could you please refactor all the manually created FRP structures here to use `define_endpoints!` instead? Then all such lines like this could be simply written as:

```
frp.output.source.pointer_style <+ model.project_name.pointer_style;
```